ui/src/app.tsx

```diff
@@ -61,25 +58,11 @@ export const App = () => {
               </Route>
               <Route path="projects" element={<ProjectsContainer />} />
               <Route path="projects/:projectId" element={<ProjectContainer />}>
-                <Route
```

Comment:
Tried out a similar approach in previous PR, but in that case I lazy loaded pages instead of just the plot component.